Formula/cayley.rb

```diff
@@ -1,56 +1,64 @@
 class Cayley < Formula
-  desc "Graph database inspired by Freebase and Knowledge Graph"
-  homepage "https://github.com/cayleygraph/cayley"
+  desc "An open-source graph database"
+  homepage "https://cayley.io/"
   url "https://github.com/cayleygraph/cayley.git",
     :tag      => "v0.7.7",
     :revision => "dcf764fef381f19ee49fad186b4e00024709f148"
+  sha256 "37ef9043cb20aeffa93d57f360fdee3e0c9d1b2fa8d404a00f3bff1b58937206"
 
   bottle do
     cellar :any_skip_relocation
     sha256 "e647be34623b1a8d635df7508f09111dfd8eb6f368a6979f9c5619b016beda8c" => :mojave
     sha256 "4177fdcb60422d484f1377e5367844ebcc9be471fffc76e717d8ad90e49ee99c" => :high_sierra
   end
 
-  depends_on "bazaar" => :build
   depends_on "go" => :build
-  depends_on "mercurial" => :build
+  depends_on "gobuffalo/tap/packr" => :build
```

Comment:
You shouldn't use personal tap in a formula in homebrew-core.

Reply:
Will it be better if I use `go get` instead?

Reply:
There's an example of `packr` being used [here](https://github.com/Homebrew/homebrew-core/blob/af1971734f76364c12f787dca92b55d1c4047be8/Formula/convox.rb).

However, it might be worthwhile creating a formula for packr in homebrew-core. The question was posed at the time when it was added to `convox`, but `convox` was the only formula in homebrew-core to need it so it wasn't done.

Reply:
Packr is already in homebrew-core.